app/controllers/idv/phone_controller.rb

```diff
@@ -76,5 +77,18 @@ def set_idv_form
     def failure_url(reason)
       idv_phone_failure_url(reason)
     end
+
+    def ensure_profile_verification_not_pending
+      verification_pending_profiles.each do |profile|
+        profile.update!(deactivation_reason: nil)
```

Comment:
How about we add a deactivation reason here, like `verification_cancelled` or something like that?